packages/pieces/community/mautic/src/lib/triggers/index.ts

```diff
@@ -0,0 +1,453 @@
+          "object": "lead",
+          "is_fixed": "1",
+          "value": null
+        },
+        "website": {
+          "id": "20",
+          "label": "Website",
+          "alias": "website",
+          "type": "url",
+          "group": "core",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "boolean": {
+          "id": "43",
+          "label": "Boolean",
+          "alias": "boolean",
+          "type": "boolean",
+          "group": "core",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "multiple_contact": {
+          "id": "44",
+          "label": "Multiple Contact",
+          "alias": "multiple_contact",
+          "type": "multiselect",
+          "group": "core",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        }
+      },
+      "social": {
+        "facebook": {
+          "id": "21",
+          "label": "Facebook",
+          "alias": "facebook",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "foursquare": {
+          "id": "22",
+          "label": "Foursquare",
+          "alias": "foursquare",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "instagram": {
+          "id": "24",
+          "label": "Instagram",
+          "alias": "instagram",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "linkedin": {
+          "id": "25",
+          "label": "LinkedIn",
+          "alias": "linkedin",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "skype": {
+          "id": "26",
+          "label": "Skype",
+          "alias": "skype",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "twitter": {
+          "id": "27",
+          "label": "Twitter",
+          "alias": "twitter",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        }
+      },
+      "personal": [],
+      "professional": []
+    }
+  },
+  "channel": "email",
+  "old_status": "contactable",
+  "new_status": "manual",
+  "timestamp": "2017-12-01T00:05:18-06:00"
+}
+
+export const triggers = [
+  {
+    name: "lead_company_change",
+    displayName: "Contact Updated",
+    description: "Triggered when a contact is updated.",
+    sampleData: {
+      "mautic.lead_company_change": [contactTestData]
+    },
+    eventType: "mautic.lead_company_change",
```

Comment:
can you please revalidate eventType for the contact updated trigger? 

docs : https://developer.mautic.org/#list-available-webhook-triggers

Reply:
Sure